Makefile

```diff
@@ -41,6 +41,13 @@ metrics:
 integration: ginkgo
 	./ginkgo -v -focus "${FOCUS}" ./integration/docker/ -- -runtime=${RUNTIME} -timeout ${TIMEOUT}
 
+kubernetes:
```

Comment:
I think you should add this to `check` target

Reply:
fixed